### PR TITLE
Feature/433035 parameter maintenace update final parameters

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/EPR.Calculator.Frontend.UnitTests/Mocks/MockData.cs [hunk 1]
```diff
@@ -19,6 +19,11 @@ public static IEnumerable<SchemeParameterTemplateValue> GetSchemeParameterTempla
                 new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-ST", ParameterValue = "2210.00" },
                 new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-WD", ParameterValue = "2210.00" },
                 new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-OT", ParameterValue = "0.00" },
+                new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-UK", ParameterValue = "250.00" },
+                new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-ENG", ParameterValue = "250.00" },
+                new SchemeParameterTemplateValue { ParameterUniqueReferenceId = "COMC-COMC-WLS", ParameterValue = "250.00" },
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.Frontend.UnitTests/Mocks/MockData.cs:24]: The unique reference should be `COMC-WLS`

File: src/EPR.Calculator.Frontend.UnitTests/Mocks/MockData.cs [hunk 2]
```diff
@@ -60,14 +61,19 @@ public static string GetSchemeParametersFileContent()
...
-                "COMC-AL,Communication costs, Aluminium,£0.00,£999,999,999.99,£2,210.45,1.Update Parameter_Values,,,",
-                "COMC - FC,Communication costs, Fibre composite,£0.00,£999,999,999.99,£2,210.00,2.File,,,",
-                "COMC - GL,Communication costs, Glass,£0.00,£999,999,999.99,£2,210.00,3.Save a Copy,,",
-                "COMC-PC,Communication costs, Paper or card,£0.00,£999,999,999.99,£2,210.00,4.Save as type:  CSV UTF-8(Comma delimited)(*.csv),,",
-                "COMC - PL,Communication costs, Plastic,£0.00,£999,999,999.99,£2,210.00,,,",
-                "COMC - ST,Communication costs, Steel,£0.00,£999,999,999.99,£2,210.00,,,",
-                "COMC - WD,Communication costs, Wood,£0.00,£999,999,999.99,£2,210.00,,,",
-                "COMC - OT,Communication costs, Other,£0.00,£999,999,999.99,£0.00,,,",
+                "COMC-AL,Communication costs by material, Aluminium,£0.00,£999,999,999.99,£2,210.45,1.Update Parameter_Values,,,",
+                "COMC - FC,Communication costs by material, Fibre composite,£0.00,£999,999,999.99,£2,210.00,2.File,,,",
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.Frontend.UnitTests/Mocks/MockData.cs:65]: There should not be a space in the unique reference.

**Review by johnbenjamin-defra — CHANGES_REQUESTED** [review]
Couple of minor changes requested

File: src/EPR.Calculator.Frontend/Controllers/UploadFileController.cs [hunk 3]
```diff
@@ -76,7 +76,7 @@ public IActionResult DownloadCsvTemplate()
             try
             {
                 var filePath = Path.Combine(Directory.GetCurrentDirectory(), StaticHelpers.Path);
-                return PhysicalFile(filePath, "text/csv", "SchemeParameterTemplate.v0.1.xlsx");
+                return PhysicalFile(filePath, "text/csv", StaticHelpers.Path);
```
Comment (udaydefra) [inline src/EPR.Calculator.Frontend/Controllers/UploadFileController.cs:79]: Similar to the constant` MediaType = "application/json"`
Please create a constant for `"text/csv"`

File: src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs [hunk 4]
```diff
@@ -5,7 +5,7 @@ public class StaticHelpers
         public const string FileNotSelected = "Please select a file";
         public const string FileMustBeCSV = "Your file must have the extension .csv";
         public const string FileNotExceed50KB = "The CSV must be smaller than 50KB";
-        public const string Path = @"wwwroot/assets/SchemeParameterTemplates/SchemeParameterTemplate.v0.1.xlsx";
+        public const string Path = @"wwwroot/assets/SchemeParameterTemplates/SchemeParameterTemplate.v1.1.xlsx";
```
Comment (udaydefra) [inline src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs:8]: Can this be a configuration item?
`SchemeParameterTemplate.v1.1.xlsx` @johnbenjamin-defra @Skumarhpa

Comment (johnbenjamin-defra) [reply src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs:8]: Hi @udaydefra, if we are going to host the template file on Azure, then we can go with the configuration item, as we can avoid doing a release whenever the template file gets modified. If not, then I think constant should be fine. Thanks.

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by johnbenjamin-defra — DISMISSED** [review]

**Review by udaydefra — APPROVED** [review]

**Review by johnbenjamin-defra — DISMISSED** [review]

**Review by udaydefra — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by rekhadhankhar2018 — APPROVED** [review]

**Pull request merged**